bin/dsn.cmake

```diff
@@ -267,19 +267,27 @@ function(dsn_setup_system_libs)
 
     set(DSN_SYSTEM_LIBS "")
 
-    find_package(RT REQUIRED)
-    set(DSN_SYSTEM_LIBS ${DSN_SYSTEM_LIBS} ${RT_LIBRARIES})
+    if (NOT APPLE)
+        find_package(RT REQUIRED)
+        set(DSN_SYSTEM_LIBS ${DSN_SYSTEM_LIBS} ${RT_LIBRARIES})
+    endif()
 
     find_package(DL REQUIRED)
     set(DSN_SYSTEM_LIBS ${DSN_SYSTEM_LIBS} ${DL_LIBRARIES})
 
     # for md5 calculation
     find_package(OpenSSL REQUIRED)
     set(DSN_SYSTEM_LIBS ${DSN_SYSTEM_LIBS} ${OPENSSL_CRYPTO_LIBRARY})
+    if (APPLE)
+        include_directories(SYSTEM ${OPENSSL_ROOT_DIR}/include)
+        link_directories("${OPENSSL_ROOT_DIR}/lib")
+    endif()
 
-    if(ENABLE_GPERF)
-        set(DSN_SYSTEM_LIBS ${DSN_SYSTEM_LIBS} tcmalloc_and_profiler)
-        add_definitions(-DDSN_ENABLE_GPERF)
+    if (NOT APPLE)
```

Comment:
Is better defining `MACOS`

Reply:
+1

Reply:
`APPLE ` is a variable in cmake, https://cmake.org/cmake/help/v3.1/variable/APPLE.html
I didn't find where is `MACOS` defined.

Reply:
Well, it's ok